### DISCUSSION TITLE
同步失败 Invalid response: 405 Method Not Allowed

### DISCUSSION THREAD

**Discussion by weizj810** [remotely-save/remotely-save#754] [General]
最近同步频繁失败，编辑完一个文档选择同步，如果在插件设置-进阶设置-处理冲突中选择保留最后修改的版本（默认）会把当前编辑的文档清空然后向服务器同步一个空白文档，选择保留文件体积较大的版本，则会同步失败并报以下错误
![image](https://github.com/user-attachments/assets/3df6d409-0131-422b-88ef-e569b1f49951)
请问有没有跟我一样出现这种情况的，求教下如何解决

**Comment by fyears** [comment]
请导出一下sync plan.有没有其他插件在同步修改文件？此外可以考虑用 smart conflict

**Reply by weizj810** [reply]
[sync_plans_hist_exported_on_1721317223598.md](https://github.com/user-attachments/files/16285580/sync_plans_hist_exported_on_1721317223598.md)
导出了两次情况的sync plan，非常感谢！之前尝试过把可能修改文件的linter和easy typing插件卸载，但是没有效果

**Reply by weizj810** [reply]
试用了一下smart conflict后依然出现Invalid response: 405 Method Not Allowed，再附上一下sync plan
[sync_plans_hist_exported_on_1721317963831.md](https://github.com/user-attachments/files/16285702/sync_plans_hist_exported_on_1721317963831.md)

**Comment by fyears** [comment]
我建议你先检查一下你自己的 webdav server 有没有配正确。特别地上传一个文件后覆盖上传是否可以。这里看起来很正常，基本很可能是服务器配置问题

**Reply by fyears** [reply]
请更新到 0.5.16 然后用设置里的 check connection 触发一下，从而看看整个连接是否正常。

**Reply by weizj810** [reply]
非常感谢，应该不是插件的问题，前两天尝试把koorf挂载的OneDrive解绑后再绑定，备份就正常了，原因应该是koofr不稳定，无法上传到OneDrive，但是check connection还是一切正常的。
